libvast/builtins/operators/pseudonymize.cpp

```diff
@@ -179,6 +181,65 @@ class plugin final : public virtual pipeline_operator_plugin {
     }
     return std::make_unique<pseudonymize_operator>(std::move(*config));
   }
+
+  [[nodiscard]] std::pair<std::string_view,
+                          caf::expected<std::unique_ptr<pipeline_operator>>>
+  make_pipeline_operator(std::string_view pipeline) const override {
+    using parsers::end_of_pipeline_operator, parsers::required_ws,
+      parsers::optional_ws, parsers::extractor, parsers::extractor_list;
+    const auto* f = pipeline.begin();
+    const auto* const l = pipeline.end();
+    const auto options = option_set_parser{{{"method", 'm'}, {"seed", 's'}}};
+    const auto option_parser = required_ws >> options;
+    auto parsed_options = std::unordered_map<std::string, data>{};
+    if (!option_parser(f, l, parsed_options)) {
+      return {
+        std::string_view{f, l},
+        caf::make_error(ec::syntax_error, fmt::format("failed to parse "
+                                                      "pseudonymize "
+                                                      "operator options: '{}'",
+                                                      pipeline)),
+      };
+    }
+    const auto extractor_parser
+      = extractor_list >> optional_ws >> end_of_pipeline_operator;
+    auto parsed_extractors = std::vector<std::string>{};
+    if (!extractor_parser(f, l, parsed_extractors)) {
+      return {
+        std::string_view{f, l},
+        caf::make_error(ec::syntax_error, fmt::format("failed to parse "
+                                                      "pseudonymize "
+                                                      "operator extractor: "
+                                                      "'{}'",
+                                                      pipeline)),
+      };
+    }
+    auto config = configuration{};
+    config.fields = parsed_extractors;
```

Comment:
we could move assign there = std::move(parsed_extractors)

Reply:
Implemented in https://github.com/tenzir/vast/pull/2877/commits/8545ed2393233c0027fb625c12d7fe3f30f9d6ca